packages/nextjs/pages/api/og.tsx

```diff
@@ -0,0 +1,220 @@
+                stroke-width="2.13331"
+              />
+              <ellipse
+                cx="136.945"
+                cy="41.1267"
+                rx="2.33222"
+                ry="2.28708"
+                fill="#B091FB"
+                stroke="#B091FB"
+                stroke-width="2.13331"
+              />
+              <path
+                d="M98.1364 48.6451C95.1776 45.0501 94.8638 37.5042 95.172 34.2088L80.4954 39.6013C79.879 44.6943 81.9748 55.9587 91.591 67.4628C103.611 81.8429 124.878 84.5391 146.145 77.3491C167.411 70.159 170.185 42.2976 169.261 39.6013C168.521 37.4443 160.391 34.3227 156.384 32.8248C156.384 34.6223 154.565 38.5781 155.305 42.1732C156.23 46.6669 153.868 51.1622 147.395 52.9597C140.923 54.7572 136.073 49.5439 131.45 48.6451C126.827 47.7463 120.788 48.6451 113.238 52.9597C106.402 56.8659 101.835 53.1389 98.1364 48.6451Z"
+                fill="#B091FB"
+              />
+              <path
+                d="M157.24 15.7188C151.692 8.52876 140.696 3.78016 131.45 2.88141C111.108 1.08389 100.72 9.04262 95.172 14.4351C89.9007 19.5588 85.1186 27.0187 86.9679 25.2212C88.8172 23.4237 113.802 16.645 125.822 16.645C140.204 16.645 155.083 21.9258 162.788 24.3224C162.48 23.1241 161.679 21.4709 157.24 15.7188Z"
+                fill="#B091FB"
+              />
+              <rect
+                x="188.284"
+                y="10.4926"
+                width="10.0771"
+                height="20.5543"
+                rx="1.52379"
+                transform="rotate(39.5223 188.284 10.4926)"
+                fill="#B091FB"
+              />
+              <rect
+                x="171.517"
+                y="20.2609"
+                width="22.8414"
+                height="3.52359"
+                rx="0.761897"
+                transform="rotate(39.5223 171.517 20.2609)"
+                fill="#B091FB"
+              />
+              <rect
+                x="174.229"
+                y="26.7572"
+                width="11.0603"
+                height="23.3793"
+                transform="rotate(39.5223 174.229 26.7572)"
+                stroke="#B091FB"
+                stroke-width="3.04759"
+              />
+              <path
+                d="M68.3263 52.1085L74.1363 104.839C74.2981 106.308 75.5388 107.419 77.0161 107.419H172.359C173.817 107.419 175.048 106.336 175.233 104.89L181.982 52.1045C182.201 50.3918 180.893 48.869 179.167 48.8443C143.452 48.3349 85.8644 48.7825 71.189 48.8773C69.4633 48.8884 68.1373 50.3932 68.3263 52.1085Z"
+                fill="#B091FB"
+              />
+              <circle cx="125.892" cy="78.8388" r="8.9888" fill="white" />
+            </svg>
+          </div>
+        </div>
+        <div tw="flex flex-col py-20 justify-center items-center text-[#551d98] w-3/5 bg-purple-100 ">
+          <div tw="mb-10 p-3 px-5 rounded-xl text-4xl bg-purple-200">{networkName}</div>
+          <div tw="mb-10 p-3 px-5 rounded-xl font-bold text-7xl text-center">{contractName}</div>
```

Comment:
On very large `contractName` where it is going to 2 lines, it was centered to the left. Added `text-center` to fix.

Example: http://localhost:3000/api/og?contractAddress=0xb153fb3d196a8eb25522705560ac152eeec57901&network=10

Pre:
![image](https://github.com/BuidlGuidl/abi.ninja/assets/55535804/977e2593-4a7f-4950-b435-a6c710856611)

Post:
![image](https://github.com/BuidlGuidl/abi.ninja/assets/55535804/43bc6805-f49f-4bbb-a1de-53dbd641c00d)